index.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+entries:
+  op-scim:
+  - apiVersion: v2
+    appVersion: v1.4.3
+    created: "2020-08-04T13:28:51.40938037-06:00"
+    description: The 1Password SCIM bridge
+    digest: 90c47a74ec4c95ab4bb76d3e97d7687ecd0a13b37b277f5b2ecb417d9d51c198
+    home: https://support.1password.com/scim
+    icon: https://1password.com/img/logo-v1.svg
+    keywords:
+    - azure active directory
+    - okta
+    - provisioning
+    - SCIM
+    - 1Password
+    maintainers:
+    - email: support+business@1password.com
+      name: 1Password Platform Integrations Team
+    name: op-scim
+    urls:
+    - op-scim-1.4.3.tgz
```

Comment:
How was this archive constructed? Just `tar -czvf . ./op-scim-1.4.3.tgz` or something else?

Reply:
To produce the tgz, you run `helm package op-scim-bridge`. This will be documented shortly.

Reply:
When we release a new version, we will update `/op-scim-bridge/chart.yml` and `/op-scim-bridge/values.yml`. Then run `helm package op-scim-bridge` followed by `helm repo index op-scim-bridge`. We'll need to tag each version with a tag that mirrors the op-scim repo before we attempt to deploy the app. I would not keep old versions in the repo itself, but allow the tags to provide this info.